tests/testthat/test-client.R

```diff
@@ -13,3 +13,30 @@ test_that("Specified connection works", {
     expect_equal(wfs$getUrl(),
                  "https://ows.emodnet-bathymetry.eu/wfs")
 })
+
+test_that("Services down handled", {
+
+    test_url <- "https://demo.geo-solutions.it/geoserver/ows?request=GetCapabilities"
+
+    webmockr::httr_mock()
+
+    z <- webmockr::stub_request('get', uri = test_url) %>%
+         webmockr::wi_th(headers = list('Accept' = 'application/json, text/xml, application/xml, */*'))
+
+    webmockr::to_return(z, status = 500)
```

Comment:
you could keep using the pipe I think :-)

Reply:
I basically followed https://docs.ropensci.org/webmockr/#many-requests-in-a-row 

I did try using the pipe but didn't work (maybe cause the second time you call `to_return() ` you are feeding the result of the first `to_return()`, an not a `stub_request()`). But I can give it another time for clarity, or just not use the pipe at all